src/components/UI/Routine.js.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router'
+import { Modal, Button } from 'react-bootstrap'
+
+class Routine extends Component {
```

Comment:
@pwentz is routine the same as exercise? We discussed yesterday that exercise would maybe a better approach, I just want to make sure we use the same language in the back and front-end of the app (even if the title to the user is different)